src/utils/dbg.js

```diff
@@ -52,11 +51,9 @@ function _formatPausePoints(dbg, url) {
 
 export function setupHelper(obj) {
   const selectors = bindSelectors(obj);
-  const actions = bindActionCreators(obj.actions, obj.store.dispatch);
```

Comment:
The actions here are already bound in https://github.com/devtools-html/debugger.html/blob/1d24d9875d7fa4d75804a5709f7afdbc4001fd28/src/utils/bootstrap.js#L55 and binding them again causes value-returning actions like `getMappedExpression` to try to re-dispatch their return value.